tests/scenarios/conftest.py

```diff
@@ -39,7 +39,7 @@ def env_scenario(request):
         env.reset()
         return env, scenario
 
-    env = holodeck.make(scenario, show_viewport=False)
-    env.reset()
-    envs[scenario] = env
-    return env, scenario
+    with holodeck.make(scenario, show_viewport=False) as env:
```

Comment:
Similar comments as above. Is this still caching them in the `envs` dict, or is it recreating it for each test? The point of putting things in conftest.py is so that we initialize as few environments as possible

Reply:
Looks like we could use a modified version of [strategy above](https://github.com/BYU-PCCL/holodeck/pull/416/files/e1ede116a20240f4c629512a879de8464de8447b#r424810628.).